plugin/controller/lib/impl/mcp/MCPControllerRegister.ts

```
@@ -300,7 +319,7 @@
     // }
     const self = this;
     const initHandler = async (ctx: Context) => {
-      const transport = new SSEServerTransport(self.mcpConfig.sseMessagePath, ctx.res);
+      const transport = new InnerSSEServerTransport(self.mcpConfig.sseMessagePath, ctx.res);
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)